tests/api-mongo.test.js

```diff
@@ -590,5 +590,27 @@ describe('MongoDB Words', () => {
           done();
         });
     });
+
+    it('should return word information when searched with wordClass filter', (done) => {
+      const keyword = 'biko';
+      const wordClass = "V"
+      getWordsFilteredByWordClass(wordClass,{ keyword })
+        .end((_, res) => {
+          expect(res.status).to.equal(200)
```

Comment:
Take a look at similar `GET` word tests and see what they check beyond the response status code. The response status code could be `200` but the body might not be what you expect, so you want to check for that.

Reply:
@Robertito1 you want to make sure that you are checking more than the status code